Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleStrokeView.m

```diff
@@ -79,7 +81,9 @@ - (void)setCenter:(CGPoint)center
 
 - (void)updateLayers
 {
-    OWSAssert(self.shapeLayer);
+    if (!self.shapeLayer) {
```

Comment:
As I read it, `shapeLayer` is set in `init` and never unset.

Why do we have nil checks for self.shapeLayer sprinkled throughout this file?

Reply:
Because the frame/bounds/center can be changed in the super constructor before `init` completes.